graphql/integration-atomic_test.go

```diff
@@ -0,0 +1,127 @@
+package graphql_test
```

Comment:
Is there a reason for putting this in a different package?

Reply:
bc the original integration test file has this as the package